java/src/main/java/vw/learner/VWBase.java

```diff
@@ -69,12 +71,15 @@ public boolean equals(Object o) {
 
         VWBase vwBase = (VWBase) o;
 
-        return nativePointer == vwBase.nativePointer;
+        if (nativePointer != vwBase.nativePointer) return false;
```

Comment:
Both `equals` and `hashCode` are methods generated by my Java editor, IntelliJ.  Because of this I'd rather not touch them.  They had to be regenerated because a new member variable was added, `predictionFunctionPointer`.